client/src/components/Tea/TeaCollectionTable/TeaCollectionTableContainer.jsx

```diff
@@ -24,28 +25,22 @@ export class TeaCollectionTableContainer extends React.Component {
     { colName: "servings", colTitle: "Servings" }
   ];
 
-  datalist = () => (
-    <DataList
-      id="fcriteria"
-      options={this.state.sortedIDs}
-      optionalArgs={this.state.formControls.filterCategory}
-      processOptions={this.getOptionsFromTeas}
-    />
-  );
+  datalist = () => <DataList id="fcriteria" options={this.state.dataList} />;
```

Comment:
Get rid of this instance method and inline the `<DataList>` into the `datalist` prop below.